src/decoders/arw.rs

```diff
@@ -45,6 +45,14 @@ impl<'a> Decoder for ArwDecoder<'a> {
     };
     let src = &self.buffer[offset..];
 
+    // Lookup the orientation of the image to store for rotating after demosaic
```

Comment:
I will refactor these into a separate function. I found out now that Canon has a tag called "Camera Orientation" that supports: Horizontal (normal), 1 = Rotate 90 CW and 2 = Rotate 270 CW. I assume they are always the same as Orientation and I guess they are part of the maker note. Should vendor specific ones take precedence or be used only if the standard orientation tag isn't available? I don't know if other vendors have similar tags.